x-pack/plugins/transform/public/app/sections/transform_management/components/stats_bar/stats_bar.tsx

```diff
@@ -29,7 +31,9 @@ interface StatsBarProps {
 }
 
 export const StatsBar: FC<StatsBarProps> = ({ stats, dataTestSub }) => {
-  const statsList = Object.keys(stats).map((k) => stats[k as StatsKey]);
+  const statsList = Object.keys(stats)
```

Comment:
Not that this is new code, but this looks like `Object.values` would do the same thing.

Reply:
Fixed here [`735d681` (#165886)](https://github.com/elastic/kibana/pull/165886/commits/735d6815a8e08f177371af3484f6975ae74fddab)